src/components/TodoList/TodoList.tsx

```diff
@@ -1,100 +1,96 @@
 import React from 'react';
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+import { SortBy } from '../../types/SortBy';
 
-export const TodoList: React.FC = () => (
-  <table className="table is-narrow is-fullwidth">
-    <thead>
-      <tr>
-        <th>#</th>
-        <th>
-          <span className="icon">
-            <i className="fas fa-check" />
-          </span>
-        </th>
-        <th>Title</th>
-        <th> </th>
-      </tr>
-    </thead>
+type ListOfTodos = {
+  listOfTodos: Todo[],
+  onSelect: (todo: Todo) => void;
+  sortBy: SortBy,
+  query: string,
+};
 
-    <tbody>
-      <tr data-cy="todo" className="">
-        <td className="is-vcentered">1</td>
-        <td className="is-vcentered" />
-        <td className="is-vcentered is-expanded">
-          <p className="has-text-danger">delectus aut autem</p>
-        </td>
-        <td className="has-text-right is-vcentered">
-          <button data-cy="selectButton" className="button" type="button">
-            <span className="icon">
-              <i className="far fa-eye" />
-            </span>
-          </button>
-        </td>
-      </tr>
-      <tr data-cy="todo" className="has-background-info-light">
-        <td className="is-vcentered">2</td>
-        <td className="is-vcentered" />
-        <td className="is-vcentered is-expanded">
-          <p className="has-text-danger">quis ut nam facilis et officia qui</p>
-        </td>
-        <td className="has-text-right is-vcentered">
-          <button data-cy="selectButton" className="button" type="button">
-            <span className="icon">
-              <i className="far fa-eye-slash" />
-            </span>
-          </button>
-        </td>
-      </tr>
+export const TodoList: React.FC<ListOfTodos> = ({
+  listOfTodos,
+  onSelect,
+  sortBy,
+  query,
```

Comment:
do not pass such values here
`TodoList` component shouldn't care about `query` and `sortBy` values (single responsibility principle). Pass the prepared array of todos (which should be visible here), i.e. filter the todos array in the App.tsx